Common subdirectories: st-1.9/docs and st-1.9.patch/docs
Common subdirectories: st-1.9/examples and st-1.9.patch/examples
Common subdirectories: st-1.9/extensions and st-1.9.patch/extensions
diff -u st-1.9/Makefile st-1.9.patch/Makefile
--- st-1.9/Makefile	2009-10-02 06:55:03.000000000 +0800
+++ st-1.9.patch/Makefile	2014-02-28 23:38:39.298697732 +0800
@@ -57,10 +57,10 @@
 # possible compilation options.
 ##########################
 
-CC          = cc
-AR          = ar
-LD          = ld
-RANLIB      = ranlib
+CC          ?= cc
+AR          ?= ar
+LD          ?= ld
+RANLIB      ?= ranlib
 LN          = ln
 
 SHELL       = /bin/sh
